central/complianceoperator/v2/compliancemanager/manager_impl.go

```diff
@@ -240,11 +240,51 @@ func (m *managerImpl) HandleScanRequestResponse(ctx context.Context, requestID s
 	return nil
 }
 
-func (m *managerImpl) ProcessRescanRequest(_ context.Context, _ interface{}) error {
-	// TODO(ROX-18091):
+func (m *managerImpl) ProcessRescanRequest(ctx context.Context, scanID string) error {
+	if !features.ComplianceEnhancements.Enabled() {
+		return errors.Errorf("Compliance is disabled. Cannot run compliance scan for configuration with ID %s", scanID)
+	}
+
 	// 1. Validate config exists in database
-	// 2. Push request to Sensor
-	panic("implement me")
+	scanConfig, found, err := m.scanSettingDS.GetScanConfiguration(ctx, scanID)
+	if err != nil {
+		return errors.Errorf("Encountered error attempting to find scan configuration with ID: %s", scanID)
+	} else if !found {
+		return errors.Errorf("Failed to find scan configuration by ID: %s", scanID)
+	}
+
+	// 2. Find applicable clusters for the given scan configuration (or just broadcast?)
```

Comment:
Since #9132 is in, the clusters are now also present in the configuration itself.  So we shouldn't need to get the clusters separately, that should be in `scanConfig.GetClusters()` or similar.